index.html

```diff
@@ -32,7 +32,7 @@
               Dropdown
             </a>
             <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <li><a class="dropdown-item" href="#">Action</a></li>
+              <li><a class="dropdown-item" href="#">Git Actions (CI)</a></li>

```

Comment:
Hi Joan!

Hope you are doing well!! 

Do you agree to add Git Actions to the dropdown? @JoanEsquivel 

Let my know, please and if I apply the change!! 🥇

Reply:
Hello Moni, 

For next iterations, please assign the reviewer(in this case I did it). 

Hope you can check what I have suggested. 

Joan.